src/apps/config/support.lua

```diff
@@ -119,12 +128,14 @@ local function record_mutable_objects_embedded_in_app_initarg (name, obj, accum)
    visit(obj)
 end
 
--- Return "in-place dependencies": a table mapping mutable object ->
--- list of app names.
-local function compute_mutable_objects_embedded_in_app_initargs (app_graph)
+-- Takes a table of follower ids (app_graph_map) and returns
+-- "in-place dependencies": a table mapping mutable object -> list of app names
```

Comment:
I think we probably need to return a table like `{follower_id => {mutable_object => {app_name...}}}` or something.  Or, `{mutable_object=>{{follower=follower_id, app_names={...}}...}}`.  I mean, an app name is ambiguous -- it could be you only mean to refer to app A in follower M, not A in follower N.